templates/csv.yaml

```diff
@@ -64,6 +78,15 @@ spec:
         displayName: Version
         path: version
       version: v1alpha1
+    - description: Represents an installation of a particular version of Knative Kafka components
+      displayName: Knative Kafka
+      kind: KnativeKafka
+      name: knativekafkas.operator.serverless.openshift.io
+      statusDescriptors:
+      - description: The version of Knative Kafka installed
+        displayName: Version
+        path: version
```

Comment:
Have we added that? I recall a discussion where we did decide not to add it.

Reply:
+1 good point. gotta remove that.

Reply:
I removed this